hydra-plutus/src/Hydra/ContractSM.hs

```diff
@@ -160,7 +159,7 @@ data InitParams = InitParams
   }
   deriving (Generic, FromJSON, ToJSON)
 
-setup :: Contract () (BlockchainActions .\/ Endpoint "init" InitParams) HydraPlutusError ()
+setup :: Contract () (Endpoint "init" InitParams) HydraPlutusError ()
```

Comment:
Not that I will regret it, but why remove `BlockchainActions` ? Just out of curiosity...